packages/main/src/components/DynamicPage/DynamicPage.jss.ts

```diff
@@ -0,0 +1,162 @@
+import { ThemingParameters } from '@ui5/webcomponents-react-base/lib/ThemingParameters';
+import { sapUiResponsiveContentPadding } from '@ui5/webcomponents-react-base/lib/spacing';
+
+export const DynamicPageCssVariables = {
+  anchorFloat: '--_ui5wcr_ObjectPage_actions_float',
+  anchorLeft: '--_ui5wcr_ObjectPage_actions_left',
+  anchorRight: '--_ui5wcr_ObjectPage_actions_right',
+  avatarMargin: '--_ui5wcr_ObjectPage_avatar_margin',
+  lastSectionMargin: '--_ui5wcr_ObjectPage_last_section_margin_bottom'
+};
```

Comment:
It would be better to rename the css variables to use `DynamicPage` instead of `ObjectPage`. Otherwise this might collide with the ObjectPage variables.